x/gamm/pool-models/stableswap/msgs.go

```diff
@@ -58,6 +58,14 @@ func (msg MsgCreateStableswapPool) ValidateBasic() error {
 		return types.ErrInvalidScalingFactors
 	}
 
+	// iterates len(msg.InitialPoolLiquidity) < 10 times
```

Comment:
Can we cover this with test please?

Reply:
This was a note that I intended to remove, not sure why it's still there (the number of iterations is capped at the max number of pool assets by definition)